convention.md

```diff
@@ -320,6 +320,12 @@ Any other topic is not part of the Homie convention.
 
 ## Extensions
 
-In addition to homie-core the device may support multiple extensions which are defined in separate conventions.
-Every extension is identified by a unique id.
-The id consists of the reverse domain name and a freely choosen prefix. For example a company `example.org` that likes to extend homie a cool feature would choose `org.example.cool-feature`. The prefix `homie` is reserved.
+This convention only covers discoverability of devices and its capabilities.
+The aim is to have standardized MQTT topics for all kind of complex scenarios.
+A Homie device may therefore support extensions, defined in separate documents.
+Every extension is identified by a unique ID and will be linked from the section.
+
+The ID consists of the reverse domain name and a freely chosen prefix.
```

Comment:
I know I wrote the original, but noticed it just now.
Isn't this a postfix not a prefix ?

Reply:
suffix is the word you are looking for ^^